### PR TITLE
Updated Czech translations

### PR DESCRIPTION
**Pull request by mipa87**

### CODE REVIEW

File: strings.txt [hunk 1]
```diff
@@ -399,22 +446,8 @@ PLUGIN_SPOTTY_SINGLES
 	RU	Синглы и EP
 	SV	Singlar och EP-skivor
 
-PLUGIN_SPOTTY_TOP_TRACKS
```
Comment (michaelherger) [inline strings.txt:402]: Hah! Good catch!

File: strings.txt [hunk 2]
```diff
@@ -631,31 +681,38 @@ PLUGIN_SPOTTY_CLIENT_ID
...
+	CS	<li>Přejděte na <a href="https://developer.spotify.com/dashboard/applications" target="_blank">Spotify Developer Portal</a> a v případě potřeby se přihlaste.</li>
+	CS	<li>Klikněte na možnost "CREATE AN APP" a postupujte podle pokynů.</li>
+	CS	<li>Zkopírujte nově vytvořené Client ID do nastavení Spotty.</li>
+	CS	<li>Pokud plánujete se Spotty používat více účtů Spotify (např. rodinný účet), budete je muset přidat do části "Users and Access" v nově vytvořené konfiguraci.</li>
+	CS	</ul>
 	DE	<div>Die Verwendung einer eigenen Client ID kann helfen, falls Sie Probleme mit dem "Rate Limiting" haben (Fehler 429). Um eine solche Client ID zu erhalten, befolgen Sie diese einfachen Schritte:</div>
 	DE	<ul style="list-style:initial">
 	DE	<li>Besuchen Sie das <a href="https://developer.spotify.com/dashboard/applications" target="_blank">Spotify Developer Portal</a>, melden Sie sich falls nötig an.</li>
-	DE	<li>Klicken Sie "Create An App", und folgen Sie den Anweisungen.</li>
+	DE	<li>Klicken Sie "CREATE AN APP", und folgen Sie den Anweisungen.</li>
```
Comment (michaelherger) [inline strings.txt:694]: I'd prefer not to use all uppercase, even if the original page does so. Could you please revert these changes for languages other than Czech?

Comment (mipa87) [reply strings.txt]: Reverted 🙂

**Comment by michaelherger** [comment]
Thanks a lot!
